server/block/cactus.go

```diff
@@ -0,0 +1,69 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/world/particle"
+	"math/rand"
+)
+
+// A Cactus is a naturally occurring block found in deserts
```

Comment:
I would remove the `A ` prefix here to be consistent with the rest of the blocks in the package.